client/src/app/plants/plant.component.html

```diff
@@ -11,4 +11,13 @@ <h3 >{{plant.cultivar}}</h3>
     <div *ngIf="rated">
         Thanks for your feedback!
     </div>
+    <br>
+    <div *ngIf="!commented">
+        <textarea [(ngModel)]="currentQuery" id="commentBox" rows="5" wrap="on" style="width: 95%;"></textarea>
```

Comment:
We should probably rename `currentQuery` to something more accurate like `currentCommentText` or just `commentText`. 

Also, do we need the `id="commentBox"` anymore?